cmd/ssh.go

```diff
@@ -31,9 +32,8 @@ var sshCmd = &cobra.Command{
 		logger := logrus.New().WithField("command", "ssh")
 
 		connectionString := viper.Get("docker.environment.KDK_USERNAME").(string) + "@localhost"
-
 		logger.Info("Connecting to KDK container")
-		sh.Command("ssh", connectionString, "-A", "-p", "2022", "-i", "~/.kdk/ssh/id_rsa", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null").SetStdin(os.Stdin).Run()
+		sh.Command("ssh", connectionString, "-A", "-p", kdk.Port, "-i", "~/.kdk/ssh/id_rsa", "-o", "StrictHostKeyChecking=no", "-o", "UserKnownHostsFile=/dev/null").SetStdin(os.Stdin).Run()
```

Comment:
Nit.  I would newline a few of these:
```
               session.Command("ssh",
                       viper.Get("docker.environment.KDK_USERNAME").(string) + "@localhost",
                       "-A",
                       "-p", "2022",
                       "-i", "~/.kdk/ssh/id_rsa",
                       "-o", "StrictHostKeyChecking=no",
                       "-o", "UserKnownHostsFile=/dev/null").SetStdin(os.Stdin).Run()
```

I'm old school and still respect 80 chars, i mean 140 chars.

Reply:
I bet we could run this all through gofmt, and be done with the formatting.

Reply:
I ran `gofmt -s -w cmd internal main.go` before opening PR.